### PR TITLE
Hosted Private Cloud powered by VMware - Lifecycle updates

### PR DESCRIPTION
**Pull request by Hubmar**

### CODE REVIEW

File: pages/cloud/private-cloud/lifecycle/guide.fr-fr.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+LL'objectif de ce guide est de rassembler les actions que vous pouvez avoir à mener dans le cadre de la gestion du cycle de vie des produits VMware hébergés chez OVHcloud. Les éléments déterminants ce cycle de vie peuvent être liés à nos opérations en datacentre, nos approvisonnements, la durée de vie des ressources physiques ou encore les versions des logiciels déployés.
+
+Dans ce guide, nous détaillerons les actions à mener pour les sujets en cours
+* Datacentre de Roubaix, Salle 29
+* Datastore antérieur à 2020
+* Host antérieur à 2018
+* Version de vSphere antérieure à vCSA 7.0 (build 20845200)
+* Version de NSX-v antérieure à 6.4.14
+
+**Nulla condimentum rhoncus nisl eget efficitur. Quisque sit amet nisl id velit vestibulum tempus nec id purus. Cras venenatis lectus ligula, id laoreet nisi vulputate id.**
```
Comment (OVH-Team-Guides) [inline pages/cloud/private-cloud/lifecycle/guide.fr-fr.md]: Hello @Hubmar, this seems to still be a work-in-progress draft.

**Comment by Hubmar** [comment]
Sorry I forgot to remove this sentence between the ** 
the rest is I believe ready to be published

File: pages/cloud/private-cloud/lifecycle_policy/guide.de-de.md [hunk 2]
```diff
@@ -39,6 +39,8 @@ Hosted Private Cloud
 * DC2014 & DC2016 (AMD PileDriver)
 * SDDC2014 & SDDC2016 (Intel Ivy Bridge, Intel Haswell)
 * SDDC2018 (Intel Broadwell, Intel Haswell)
+* Essentials (Intel Broadwell)
```
Comment (OVH-Team-Guides) [inline pages/cloud/private-cloud/lifecycle_policy/guide.de-de.md:42]: Hello @Hubmar, you added these 2 lines on the de-de sub but not on the others. Should I replicate them onto all subs ?

File: pages/cloud/private-cloud/lifecycle/guide.fr-fr.md [hunk 3]
```diff
@@ -0,0 +1,204 @@
...
+
+Si votre version de vSphere n'est pas vCSA 7.0 (build 20845200), veuillez créer un ticket afin de planifier une mise à jour vers la dernière version.
+
+#### Récupérer les informations de votre PCC dans vSphere
+
+Rendez vous dans la vue `Mise en réseau`{.action} de votre client vSphere puis dans `Sécurité`{.action} et enfin cliquez sur l'onglet `Upgrade`{.action}.
+
+Prenez note de la version de votre NSX-v.
+
+![vSphere - récupération des données](images/vsphere-version-nsx-v.png){.thumbnail}
```
Comment (OVH-Team-Guides) [inline pages/cloud/private-cloud/lifecycle/guide.fr-fr.md]: @Hubmar, please provide the vSphere screenshot.
I added the Control Panel screenshot, please check if it's valid.

**Comment by OVH-Team-Guides** [comment]
@Hubmar , I deleted the 2 screenshots you added for the Federation guide as they were no compliant with our requirements. Please contact us for more information.

File: pages/cloud/private-cloud/lifecycle/guide.fr-fr.md (hunk 3 above)
Comment (OVH-Team-Guides) [reply pages/cloud/private-cloud/lifecycle/guide.fr-fr.md]: Screenshot and valid path provided by Naïm.

File: pages/cloud/private-cloud/lifecycle_policy/guide.de-de.md (hunk 2 above)
Comment (OVH-Team-Guides) [reply pages/cloud/private-cloud/lifecycle_policy/guide.de-de.md:42]: I decided to replicate the lines.

**Comment by OVH-Team-Guides** [comment]
Thank you for your contribution @Hubmar

**Pull request merged**